process/elasticProcessor.go

```diff
@@ -227,19 +234,26 @@ func (ei *elasticProcessor) getExistingObjMap(hashes []string, index string) (ma
 }
 
 func getDecodedResponseMultiGet(response objectsMap) map[string]bool {
+	const (
```

Comment:
move them in the top of the file for performance reasons

Reply:
done.